src/infra/core/src/repos/metadata_chain_impl.rs

```diff
@@ -815,14 +755,13 @@ where
             }
         }
 
-        let data = FlatbuffersMetadataBlockSerializer
+        let block_data = FlatbuffersMetadataBlockSerializer
             .write_manifest(&block)
             .int_err()?;
-
         let res = self
-            .obj_repo
-            .insert_bytes(
-                &data,
+            .meta_block_repo
+            .insert_block_data(
```

Comment:
We provide DTO-based `get` method, so why don't we hide the serialization of a block inside `MetadataBlockRepo::insert_block()` too to make interface symmetrical?

Reply:
After [the hotfix](https://github.com/kamu-data/kamu-cli/pull/496), we don't need this structure -- we can just cache blocks, not bytes

Thus, methods working with bytes (insert, delete, get size) are no longer needed -- only block-level methods are left in `MetadataBlockRepository`